src/webhooks/index.ts

```diff
@@ -66,6 +67,40 @@ export default function (
             (event.data.object as Stripe.PaymentMethod).id,
           );
           break;
+
+        case 'setup_intent.succeeded': {
+          stripe.subscriptions
+            .create({
+              customer: (event.data.object as Stripe.SetupIntent).customer as string,
+              default_payment_method: (event.data.object as Stripe.SetupIntent).payment_method as string,
+              items: [
+                {
+                  price: (event.data.object as Stripe.SetupIntent).metadata?.priceId as string,
+                  metadata: {
+                    is_teams: 0,
+                  },
+                },
+              ],
+              expand: ['latest_invoice.payment_intent'],
+            })
+            .then(async (res) => {
```

Comment:
Do not mix `async-await` with `then` chaining

Reply:
Fixed